### PR TITLE
Copy cryptopp 8.3 formula from algorand

### PR DESCRIPTION
**Pull request by Baltoli**
It's more idiomatic for us to provide this formula in our homebrew repo, rather than as a local `.rb` file in the algorand semantics. Doing so will allow us to write something like:
```ruby
depends_on "kframework/k/cryptopp@8.3.0"
```
when packaging the algorand semantics.

### CODE REVIEW

**Review by ehildenb — APPROVED** [review]

**Comment by ehildenb** [comment]
@Baltoli I've approved so you're not blocked on merging (though I think you could have anyway, not sure), but am assuming that you'll only merge once you're at least 75% confident you got it right.

**Comment by Baltoli** [comment]
@ehildenb I'm pointing at a different copy of the formula while I prototype, so this isn't blocking me - I don't seem to have merge rights, though, so I'll ping you when I'm sure it's ready. Thanks!

**Pull request merged**
